BikePoznan/Controllers/BikeStationMapViewControler.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  BikeStationMapViewControler.swift
+//  BikePoznan
+//
+//  Created by Mateusz Gozdzik on 21/05/2022.
+//
+
+import Foundation
+import UIKit
+import MapKit
+import CoreLocation
+
+class BikeStationsMapieViewController: UIViewController, CLLocationManagerDelegate {
+
+    
+    @IBOutlet weak var Distance: UILabel!
+    @IBOutlet weak var mapKit: MKMapView!
+    var Coor: Array<Double> = []
+    var free_bikes: String = ""
+    let mangager = CLLocationManager()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        mangager.desiredAccuracy = kCLLocationAccuracyBest
+        mangager.delegate = self
+        mangager.requestWhenInUseAuthorization()
+        mangager.startUpdatingLocation()
+    }
+
+    func locationManager(_ manager: CLLocationManager, didUpdateLocations locations: [CLLocation]) {
+ 
+     
+        let locValue:CLLocationCoordinate2D = manager.location!.coordinate
+        print("locations = \(locValue.latitude) \(locValue.longitude)")
+      
+        let location = locations[0]
+        let span = MKCoordinateSpan(latitudeDelta: 0.1, longitudeDelta: 0.1)
+        let userLocation:CLLocationCoordinate2D = CLLocationCoordinate2DMake(location.coordinate.latitude, location.coordinate.longitude)
+        
+        
+        let region = MKCoordinateRegion(center: userLocation, span: span)
+       mapKit.setRegion(region, animated: true) //#1 wyswietanie uzytkownika
+        self.mapKit.showsUserLocation = true
+        
+        let cordinate1 = Coor[0]
+        let cordinate2 = Coor[1]
+        let coordinate = CLLocationCoordinate2D(latitude: cordinate2, longitude: cordinate1)
+        
+        let pin = MKPointAnnotation()
+        pin.coordinate = coordinate
+        mapKit.addAnnotation(pin)
+        pin.title = free_bikes
```

Comment:
```suggestion
        pin.title = free_bikes
        mapKit.addAnnotation(pin)
```
Ładniej w ten sposób